assets/example.spec.js

```diff
@@ -1,411 +1,15 @@
-// @ts-check
 const { test, expect } = require('@playwright/test');
```

Comment:
```suggestion
// @ts-check
const { test, expect } = require('@playwright/test');
```
this is very important to make the JS developer's life easier.